src/graphqlHelpers.ts

```diff
@@ -891,7 +902,7 @@ export function typeScriptDefinitionObjectForFragment(
             namedFragments: [],
             inlineFragments: [],
             selections: {
-              displayedName: {
+              [displayedName]: {
```

Comment:
@sgrove this is a drive-by fix, I think you didn't intend to start with `displayedName` literally.